extension/js/common/core/mime.ts

```diff
@@ -403,4 +409,36 @@ export class Mime {
     return node;
   }
 
+  private static extractInlineImagesToAttachments = (html: string, rootNode: any) => {
```

Comment:
Looks good in high-level overview :+1: but will need to give this more scrutiny later.

Reply:
@seisvelas should scrutinize it for attack vectors if any